pages/common/nc.md

```diff
@@ -1,8 +1,8 @@
 # nc
 
-> Reads and writes tcp or udp data.
+> Netcat is a versatile utility for working with tcp or udp data.
```

Comment:
```suggestion
> Netcat is a versatile utility for working with TCP or UDP data.
```

I'd capitalize TCP and UDP here while we're at it.

Reply:
Thanks, updated.